boards/shields/adafruit_16_ch_pwm_servo/adafruit_16_ch_pwm_servo.overlay

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 Nick Ward <nix.ward@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_i2c {
+	status = "okay";
+
+	pca9685: pca9685@40 {
```

Comment:
device nodelabel should be `<device>_<shield>`, see https://docs.zephyrproject.org/latest/hardware/porting/shields.html

Reply:
Done